package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lenkedin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "workspaces": [
+    "apps/*"
```

Comment:
smart! A question here is that what if we want to create a flask or Django backend folder inside apps/? Since there will be no package.json file, will it still work in that case?

Reply:
Yes this is the caveat of the set up. If we’re going to use express then it’ll works fine otherwise we’ll need another repo and specify the server endpoint